ibis/common/exceptions.py

```diff
@@ -21,6 +21,13 @@
     from collections.abc import Callable
 
 
+class TableNotFound(Exception):
+    def __init__(self, table_name, message="Table not found"):
+        self.table_name = table_name
```

Comment:
Are we using this attribute somewhere outside the constructor here?

Reply:
I initially thought I might but it looks like I'm not.  
```suggestion
```